copter/source/docs/traditional-helicopter-tips.rst

```diff
@@ -0,0 +1,22 @@
+.. _traditional-helicopter-tips:
+
+===========================
+Traditional Helicopter Tips
+===========================
+
+- Be sure to follow the wiki instructions, step by step!
+- Be sure the CG is in line with the main rotor shaft
+- Blade tracking must be within a quarter blade width or better to avoid excessive vibration
+- Vibration is the arch enemy of Helicopter autopilots!
+   - Internal Combustion Engines and Turbines can created high frequency vibrations that do not appear in the VIBE logs and can be aliased to low frequencies which can impact EKF and Attitude control (see below for ground tests to check for this). This also can occur in electric helicopters.
+   - The autopilot should be well isolated for vibration. Using autopilots with internal IMU vibration damping and vibration reducing mounts is recommended.
+- Traditional Helicopter *is* capable of inverted flight and using full negative collective ranges.
+
+Checking for High Frequency Vibration
+=====================================
+
+Symptoms of high frequency vibration can be seeming loss of precision or control in attitude control, resulting in the helicopter beginning to roll or pitch away from level in stabilized modes. In the extreme, it can result in an :ref:`ekf-inav-failsafe`.
```

Comment:
"seeming loss of precision or control in attitude control" can you make this more clear

Reply:
reworded a bit

Reply:
Thank you